tasks/depcheck/index.js

```diff
@@ -1,31 +1,34 @@
-const {loadPackages, iter} = require('lerna-script'),
-  checkDeps = require('depcheck')
+const {loadPackages, iter} = require('lerna-script')
+const checkDeps = require('depcheck')
+const colors = require('colors')
 
 function depcheckTask({packages, depcheck} = {}) {
   return async log => {
     const lernaPackages = await (packages || loadPackages())
     log.info('depcheck', `checking dependencies for ${lernaPackages.length} modules`)
 
-    return iter.parallel(lernaPackages, {build: 'depcheck', log})((lernaPackage, log) => {
-      return checkModule(lernaPackage, log, depcheck)
+    return iter.parallel(lernaPackages, {build: 'depcheck'})(lernaPackage => {
+      return checkModule(lernaPackage, depcheck)
     })
   }
 }
 
-function checkModule(lernaPackage, log, depcheckOpts = {}) {
+function checkModule(lernaPackage, depcheckOpts = {}) {
```

Comment:
because it add it's prefixes, such as `info:....`, `WARN:...`
This is part of the "noise" I wanted to reduce
(I hate to use both logs, it looks redundant but I had no choice)

Reply:
@viliusl WDYT?

Reply:
well npm log let's say is annoying, but then you can control it via flag (https://github.com/wix/lerna-script/blob/master/lerna-script/bin/cli.js#L9) - maybe level here should be `silly` or similar, but I would not introduce separate logger (console) which you cannot control with existing options.

Reply:
@zivl WDYT?

Reply:
My goal is to print to the console a decent log of the dependencies.
I give you here the example of the clearer log printed with this change.
```
info lerna-script executing task: "deps:depcheck"
info depcheck checking dependencies for 19 modules
unused deps found for module experts-server
{
  dependencies: [
    '@wix/experts-server-api',
    '@wix/wix-errors',
    '@wix/wix-express-require-https',
    '@wix/wix-json-rpc-server',
    'body-parser',
    'express'
  ]
}
{ 
  devDependencies: [ 'npm-version-up', '@wix/wix-proto-codegen' ] 
}

unused deps found for module @wix/experts-common
{
  devDependencies: [
    '@testing-library/react',
    '@wix/experts-common-test',
    'jest',
    'react-dom',
    'wix-style-react'
  ]
}
```
using `log level` won't help since I still get the irritating prefix (irritating in this case since I might have a lot of printing)
@viliusl can you be more specific about the "cannot control" the log?

Reply:
console.log will not respect loglevel (silly, info, fatal) which is otherwise respected all around codebase.

Reply:
@viliusl would you suggest any other way I can use `npm log` without having those prefixes printed?

Reply:
@hugebdu @viliusl - where are we on this one?

Reply:
I'm fine to proceed.
@viliusl ?